yodaconvert/src/components/Converter.js

```diff
@@ -1,24 +1,39 @@
 import React, {Component} from "react";
 import '../App.css';
+import convertToHTML from 'markdown-to-html-converter' //npm install markdown-to-html-converter
 import ButtonConvDel from './Button.jsx';
 
 
 class Components extends Component {
     constructor (props) {
         super(props)
-        this.state={}
+        this.state={
+            inputText:'',
+            outputText:''
+        }
+        this.handleInputChange = this.handleInputChange.bind(this)
     }
+
+
+handleInputChange(e) {
+    this.setState({ 
+        inputText: e.target.value,
+        outputText : convertToHTML(this.state.inputText) })
+      }
```

Comment:
fix indentation, and avoid reading the state while writing it:

      handleInputChange(e) {
        const newText = e.target.value
        this.setState({ 
          inputText: newText,
          outputText : convertToHTML(newText)
        })
      }